--- SPECS/linux/linux-esx.spec	2020-04-28 22:12:54.000000000 +0200
+++ SPECS/linux/linux-esx.spec	2020-04-29 09:34:37.000000000 +0200
@@ -16,7 +16,7 @@
 Source3:        update_photon_cfg.postun
 Source4:        check_for_config_applicability.inc
 # Photon-checksum-generator kernel module
-Source5:        https://github.com/vmware/photon-checksum-generator/releases/photon-checksum-generator-%{photon_checksum_generator_version}.tar.gz
+Source5:        https://github.com/vmware/photon-checksum-generator/archive/v%{photon_checksum_generator_version}.tar.gz
 %define sha1 photon-checksum-generator=b2a0528ce733e27bf332ea533072faf73c336f0c
 Source6:        genhmac.inc
 # common
