--- SPECS/linux/linux-esx.spec	2020-04-28 22:12:54.000000000 +0200
+++ SPECS/linux/linux-esx.spec	2020-04-29 09:34:37.000000000 +0200
@@ -2,8 +2,8 @@
 %global photon_checksum_generator_version 1.0
 Summary:        Kernel
 Name:           linux-esx
-Version:        4.19.112
-Release:        5%{?kat_build:.kat}%{?dist}
+Version:        4.19.118
+Release:        0%{?kat_build:.kat}%{?dist}
 License:        GPLv2
 URL:            http://www.kernel.org/
 Group:          System Environment/Kernel